ethereum_history_api/circuits/lib/src/rlp/list.nr

```diff
@@ -0,0 +1,9 @@
+use crate::rlp::fragment::RlpFragment;
+use crate::misc::bounded_vecs::bounded_vec_from_array;
+
+type RlpList<MAX_FIELDS> = BoundedVec<RlpFragment, MAX_FIELDS>;
+
+pub fn create_rlp_list<N, MAX_FIELDS>(fragments: [RlpFragment; N]) -> RlpList<MAX_FIELDS> {
```

Comment:
This function seems to be redundant. It just calls one other function under the hood and it's used only in tests

Reply:
Removed